dataquality/clients/api.py

```diff
@@ -582,6 +589,21 @@ def get_presigned_url(
         )
         return response["url"]
 
+    def upload_image_dataset(
+        self,
+        project_id: str,
+        file_path: str,
+    ) -> None:
+        self.make_request(
+            request=RequestType.POST,
+            url=f"{config.api_url}/{Route.upload}/{project_id}"
```

Comment:
Nit-pick so feel free to ignore, but this would be more inline with other endpoints 

```suggestion
            url=f"{config.api_url}/{Route.projects}/{project_id}/{Route.upload}"
```

Reply:
Just saw the API pr, i actually disagree with what i said above, so ignore